task/queue/queue.go

```diff
@@ -19,8 +19,9 @@ type TaskQueue struct {
 }
 
 // New creates task queue which is processed by number of workers. Number of task is limited by limit.
-func New(numWorkers, limit int) (*TaskQueue, error) {
-	p, err := ants.NewPool(numWorkers, ants.WithPreAlloc(true), ants.WithNonblocking(true))
+func New(numWorkers, limit int, options ...ants.Option) (*TaskQueue, error) {
+	options = append(options, ants.WithNonblocking(true))
```

Comment:
Shouldn't we define a yaml config struct here in the kit ?

Reply:
Sure. Fixed